examples/Cassie/dispatcher_robot_in.cc

```diff
@@ -56,15 +60,25 @@ int do_main(int argc, char* argv[]) {
   // Create LCM receiver for commands
   auto command_receiver = builder.AddSystem<RobotInputReceiver>(*tree);
 
-  // auto command_sub = builder.AddSystem(
-  //     LcmSubscriberSystem::Make<dairlib::lcmt_robot_input>(channel_u,
-  //         &lcm_local));
-  // builder.Connect(*command_sub, *command_receiver);
+    // Create state estimate receiver, used for safety checks
+  auto state_sub = builder.AddSystem(
+      LcmSubscriberSystem::Make<dairlib::lcmt_robot_output>(
+          FLAGS_state_channel_name, &lcm_local));
+  auto state_receiver = builder.AddSystem<systems::RobotOutputReceiver>(*tree);
+  builder.Connect(*state_sub, *state_receiver);
+
+  double input_supervisor_update_period = 1.0/100.0;
+  auto input_supervisor = builder.AddSystem<InputSupervisor>(*tree,
+      FLAGS_max_joint_velocity, input_supervisor_update_period);
+  builder.Connect(state_receiver->get_output_port(0),
+      input_supervisor->get_input_port_state());
+  builder.Connect(command_receiver->get_output_port(0),
+      input_supervisor->get_input_port_command());
 
   // Create and connect translator
   auto input_translator =
       builder.AddSystem<systems::CassieInputTranslator>(*tree);
-  builder.Connect(*command_receiver, *input_translator);
+  builder.Connect(*input_supervisor, *input_translator);
```

Comment:
In line 96, should it be 
`builder.Connect(*input_supervisor, *net_command_sender);`
instead of 
`builder.Connect(*command_receiver, *net_command_sender);`?

The command after the guard is probably what we want to see on the Sherriff end.